ssv/runner.go

```diff
@@ -114,8 +114,11 @@ func (b *BaseRunner) baseConsensusMsgProcessing(runner Runner, msg *qbft.SignedM
 		prevDecided, _ = b.State.RunningInstance.IsDecided()
 	}
 
-	if err := b.processPreConsensusJustification(runner, b.highestDecidedSlot, msg); err != nil {
-		return false, nil, errors.Wrap(err, "invalid pre-consensus justification")
+	// TODO: revert `if false` after pre-consensus justification is fixed.
+	if false {
+		if err := b.processPreConsensusJustification(runner, b.highestDecidedSlot, msg); err != nil {
+			return false, nil, errors.Wrap(err, "invalid pre-consensus justification")
+		}
 	}
```

Comment:
nit:
why not just comment out like you did in `all_tests`? I just find the `if false` funny and weird.
The request changes is because of the other comment.